zilliqa/src/db.rs

```diff
@@ -414,8 +414,41 @@ impl Db {
             }
         }
 
+        // Helper function used for inserting entries from memory (which backs storage trie) into state_trie db (for now it's sql-lite)
```

Comment:
```suggestion
        // Helper function used for inserting entries from memory (which backs storage trie) into persistent storage
```
(so we don't have an outdated comment when we merge redb :) )